### PR TITLE
OPSEXP-2650 Update supported matrix for multiple acs versions with new adw/acc

### PR DESCRIPTION
**Pull request by pmacius**
Ref: OPSEXP-2650

### CODE REVIEW

File: deployments/values/supported-matrix.yaml [hunk 1]
```diff
@@ -239,13 +239,13 @@ matrix:
       version: "2.0.3"
       pattern: *ga_hotfixes_pattern
     search-enterprise:
-      version: "3.1.1"
+      version: "3.1"
       pattern: *ga_hotfixes_pattern
```
Comment (gionn) [inline deployments/values/supported-matrix.yaml:243]: please explain

File: deployments/values/supported-matrix.yaml [hunk 2]
```diff
@@ -131,16 +131,16 @@ matrix:
       version: *search_ga_version
       pattern: *ga_hotfixes_pattern
     search-enterprise:
-      version: "3.3.1"
+      version: "3.3"
       pattern: *ga_hotfixes_pattern
     sync:
-      version: "3.11"
+      version: "3.11.1"
```
Comment (gionn) [inline deployments/values/supported-matrix.yaml:137]: please explain

**Review by gionn — COMMENTED** [review]
hard to review without seeing a workflow run using these changes

make sure to have a more descriptive pr title

File: deployments/values/supported-matrix.yaml (hunk 1 above)
Comment (pmacius) [reply deployments/values/supported-matrix.yaml:243]: https://docs.alfresco.com/content-services/7.2/support/ It says there thats its 3.1

**Comment by pmacius** [comment]
tested on: https://github.com/Alfresco/flux-alfresco-pipeline/actions/runs/8797543154/job/24142653252

**Pull request merged**